src/Model/MongoDB/BaseDocumentManager.php

```diff
@@ -16,6 +16,8 @@
 
 /**
  * @author Hugo Briand <briand@ekino.com>
+ *
+ * @mixin DocumentManager
```

Comment:
Is that a standard phpdoc type?

Reply:
Some IDE supports them and it doesn't hurt much ;)